test/chai-extensions.js

```diff
@@ -0,0 +1,154 @@
+var chai = require('chai');
```

Comment:
It could be helpful for the plugin writers too, plus we could add tests for those assertions